kinetic/distribution.yaml

```diff
@@ -17572,5 +17573,10 @@ repositories:
       url: https://github.com/zivid/zivid-ros.git
       version: master
     status: maintained
+  swarmros:
+    doc:
+      type: git
+      url: https://github.com/amilankovich-slab/swarmros-release.git
```

Comment:
The doc url should be a branch (preferred, but tag is ok) on the upstream repository. Usually it's the same as the source entry.